virt/virt-4-11-release-notes.adoc

```diff
@@ -57,22 +57,20 @@ The SVVP Certification applies to:
 * The link for installing the Migration Toolkit for Virtualization (MTV) Operator in the {product-title} web console has been moved. It is now located in the *Related operators* section of the *Getting started resources* card on the *Virtualization* -> *Overview* page.
```

Comment:
SVVP. I'm not sure that we got certification.  [CNV-15202](https://issues.redhat.com/browse/CNV-15202).

@stu-gott Can you confirm?

Reply:
Concur with this.